tests/scale/test_load.py

```diff
@@ -159,13 +159,23 @@ def test_scaling_load(master_count,
                     range(min_index, max_index)]
     # create service accounts in parallel
     sdk_security.install_enterprise_cli()
-    service_account_threads = _spawn_threads(masters,
-                                            _create_service_accounts,
-                                            security=security_mode)
 
-    thread_failures = _wait_and_get_failures(service_account_threads,
-                                             timeout=SERVICE_ACCOUNT_TIMEOUT)
-    # launch Jenkins services
+    current = 0
+    end = max_index - min_index
+    while current + batch_size <= end:
```

Comment:
what if 
current = 10
batch = 5
end = 13

then we skip the last three?
Or it is guaranteed that this can never happen?

Reply:
yes, we will skip, although the commands that launches those scripts are aware of that.